go.mod

```diff
@@ -18,8 +18,9 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.14.3
-	github.com/lib/pq v1.5.2
 	github.com/interuss/stacktrace v0.0.0-20200827180054-b2e58cf48818
+	github.com/lib/pq v1.5.2
+	github.com/palantir/stacktrace v0.0.0-20161112013806-78658fd2d177 // indirect
```

Comment:
The palantir reference has now been removed from the codebase; this addition is no longer necessary.